client/src/legacy/LeftAndMain.js

```diff
@@ -889,12 +889,12 @@ $.entwine('ss', function($) {
       this.find('.cms-tabset,.ss-tabset').each(function(i, el) {
         var id = $(el).attr('id');
         if(!id) return; // we need a unique reference
-        if(!$(el).data('tabs')) return; // don't act on uninit'ed controls
+        if(!$(el).data('uiTabs')) return; // don't act on uninit'ed controls
```

Comment:
'tabs' was renamed to 'uiTabs' in jquery-ui 1.10 - I couldn't find a reference of where this was done, though console.log'ing `$el.data()` showed it to now be `'uiTabs'`